client/src/components/Rating/RatingStars.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faStar as EmptyStar} from '@fortawesome/free-regular-svg-icons';
+import {faStar as SolidStar}  from '@fortawesome/free-solid-svg-icons';
+import './RatingStyles.css';
+
+export const RatingStars = () => {
+  let stars = new Array(5).fill(EmptyStar);
```

Comment:
Also not sure that it's a good idea to use react components here. I think that you can use pure js object instead of

Reply:
I hope I correctly understood you, I changed it like this:
let stars = new Array(5).fill("EmptyStar");
<FontAwesomeIcon icon={starSet[star] == "EmptyStar" ? EmptyStar : SolidStar}>

Reply:
@Safwood youp! This will be better